rotkehlchen/globaldb/handler.py

```diff
@@ -627,12 +627,19 @@ def get_asset_data(
     def fetch_underlying_tokens(
             cursor: DBCursor,
             parent_token_identifier: str,
+            from_builtin_db: bool = False,
     ) -> Optional[List[UnderlyingToken]]:
         """Fetch underlying tokens for a token address if they exist"""
-        cursor.execute(
-            'SELECT B.address, B.token_kind, A.weight FROM underlying_tokens_list AS A JOIN evm_tokens as B WHERE A.identifier=B.identifier AND parent_token_entry=?;',  # noqa: E501
-            (parent_token_identifier,),
-        )
+        if from_builtin_db is True:
+            cursor.execute(
+                'SELECT B.address, B.token_kind, A.weight FROM builtin_db.underlying_tokens_list AS A JOIN builtin_db.evm_tokens as B WHERE A.identifier=B.identifier AND parent_token_entry=?;',  # noqa: E501
```

Comment:
Maybe instead of separating it into 2 completely different queries, just make one parametrizable?